package.json

```diff
@@ -5,15 +5,69 @@
     "type": "git",
     "url": "https://github.com/brownhci/WebGazer.git"
   },
-  "devDependencies": {
-    "git": "^0.1.5",
-    "grunt": "*",
-    "grunt-contrib-concat": "*",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-uglify-es": "*",
-    "jsdoc": "^3.6.3"
+  "main": "dist/webgazer.commonjs2.js",
+  "scripts": {
+    "lint": "eslint .",
+    "clean": "rimraf dist",
+    "prebuild": "npm run clean",
+    "build": "webpack --progress --colors --config webpack.config.js --mode=production",
```

Comment:
Needs to be an easy way to specify development mode as well.  Maybe add a new script `dev build`?